app/views/images/_modal.html.erb

```diff
@@ -1,11 +1,14 @@
-<div id="<%= id %>" class="modal" tabindex="-1">
-  <div class="modal-dialog modal-dialog-centered modal-sm">
+<div id="<%= id %>" class="modal" tabindex="-1" data-bs-theme="dark">
```

Comment:
This modal made more sense as always in darkmode, more of a lightbox effect, kind of?